src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--29">
+      <div class="calendar__day calendar__day--1 calendar__day--start-day_fri"></div>
+      <div class="calendar__day calendar__day--2 "></div>
+      <div class="calendar__day calendar__day--3"></div>
+      <div class="calendar__day calendar__day--4"></div>
+      <div class="calendar__day calendar__day--5"></div>
+      <div class="calendar__day calendar__day--6"></div>
+      <div class="calendar__day calendar__day--7"></div>
+      <div class="calendar__day calendar__day--8"></div>
+      <div class="calendar__day calendar__day--9"></div>
+      <div class="calendar__day calendar__day--10"></div>
```

Comment:
Why do you need modifier `calendar__day--1` and others. Try to rewrite it with `&:nth-child`.

This way you'll avoid redundant modifiers.